database_wrapper.py

```diff
@@ -289,34 +343,34 @@ def get_professor_properties(self, lastName) -> List[str]:
         """
         To get a particular professor's properties
         """
-        cursor = self.connection.cursor()
-        cursor.execute('use `{}`'.format(self.database))
+
         # FIXME: resolve unused variable `props`, until then, commented out
-        # props = get_property_from_entity(self, ["*"],
-        #                                  "Professors",
-        #                                  condition_field="lastName",
-        #                                  condition_value=lastName)
-        tups = cursor.fetchall()
-        cursor.close()
-        return [x[0] for x in tups]
+        props = self.get_property_from_entity(prop=["*"], entity="Professors", condition_field="lastName", condition_value=lastName)
+        return props
```

Comment:
Can we revert this change? `yield_entites` is meant to work as a Python generator and therefore it should not `return` anything. Rather it should `yield` in the middle of an iteration.

Reply:
Actually, I didn’t realize it was wrong before too. 
Feel free to ignore and I’ll fix it later